qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -815,9 +815,11 @@ def _on_fullscreen_requested(self, request):
         self._tab.data.fullscreen = on
         self._tab.fullscreen_requested.emit(on)
         if on:
-            notification = miscwidgets.FullscreenNotification(self._widget)
-            notification.show()
-            notification.set_timeout(3000)
+            timeout = config.instance.get('content.fullscreen.overlay_timeout')
+            if timeout is not 0:
```

Comment:
This should be `if timeout != 0`, this only works by accident with `is` - see e.g. [the StackOverflow answers in this thread](https://stackoverflow.com/questions/132988/is-there-a-difference-between-and-is).

With newer Python versions, you'll also get a warning about this:

```
SyntaxWarning: "is not" with a literal. Did you mean "!="?
```

Which is why most of the test environments are currently failing on Travis CI (see the commit status at the bottom).